localstack/services/apigateway/apigateway_listener.py

```diff
@@ -118,9 +118,12 @@ def forward_request(self, method, path, data, headers):
                     response = Response()
                     parsed_result = json.loads(result)
                     response.status_code = int(parsed_result['statusCode'])
-                    response.headers.update(parsed_result['headers'])
-                    response_body = parsed_result['body']
-                    response._content = json.dumps(response_body)
+                    response.headers.update(parsed_result.get('headers', {}))
+                    try:
+                        response_body = parsed_result['body']
```

Comment:
Here we can apply a similar pattern as for `'headers'` (and remove the `try/except):
```
response_body = parsed_result.get('body', {})
response._content = json.dumps(response_body)
```